test/evergreen_develop.yml

```diff
@@ -121,6 +121,8 @@ buildvariants:
   run_on:
   - ubuntu2004-test
   expansions:
+    database: WTCodeCoverageDB
```

Comment:
Suggest making it more general (e.g. WTCodeStatisticsDB) for the next code complexity metric. I'm okay if you prefer to cover it in the next code complexity PR.

Reply:
Fixed in 8f643b8384ce228db1a05bad5b326e67a3713d8c